opendevin/core/config.py

```diff
@@ -433,8 +433,7 @@ def load_from_toml(cfg: AppConfig, toml_file: str = 'config.toml'):
     try:
         with open(toml_file, 'r', encoding='utf-8') as toml_contents:
             toml_config = toml.load(toml_contents)
-    except FileNotFoundError as e:
-        logger.opendevin_logger.info(f'Config file not found: {e}')
+    except FileNotFoundError:
```

Comment:
QQ, if I'm running through development mode (make build -> make run) and I don't create a config.toml which I believe is required in that case, would that somehow be shown to the user? Or does another error occur?

Reply:
I think this info message was there for longer than just this refactoring, in some form. But the config toml file was optional and it should be, are you sure it's not optional anymore @mamoodi ? 

There are two aspects here: the most common, if I run with the UI, I'd set the model/key in the UI, so the config values would not be used; workspace has a default value, and well, everything else has a default value, right?

The other aspect is that vars in config.toml have env vars corresponding to them. That is, I can, if I want to, set any regular in the env, and not in toml, and that's okay. The only things I cannot set are the special named llm configs or agent configs, that's true, but the app can run.

Reply:
@enyst I didn't know the config.toml was optional in general. That's really good to know. I thought it's mandatory when following the dev workflow.

In that case, perfect! Thanks so much for removing this @tobitege